eslint.config.js

```diff
@@ -1,25 +1,24 @@
 import hypothesisBase from 'eslint-config-hypothesis/base';
 import hypothesisJSX from 'eslint-config-hypothesis/jsx';
 import hypothesisTS from 'eslint-config-hypothesis/ts';
+import { defineConfig, globalIgnores } from 'eslint/config';
 import globals from 'globals';
 
-export default [
-  {
-    ignores: [
-      '.tox/**/*',
-      '.yalc/**/*',
-      '.yarn/**/*',
-      'build/**/*',
-      '**/vendor/**/*.js',
-      '**/coverage/**/*',
-      'docs/_build/*',
-      'dev-server/static/**/*.js',
-    ],
-  },
+export default defineConfig([
+  globalIgnores([
+    '.tox/**/*',
+    '.yalc/**/*',
+    '.yarn/**/*',
+    'build/**/*',
+    '**/vendor/**/*.js',
+    '**/coverage/**/*',
+    'docs/_build/*',
+    'dev-server/static/**/*.js',
+  ]),
```

Comment:
This is equivalent to the previous `{ ignores: [...] }`. However, if another property is added by mistake to that object, then those ignores become local to files affected by that specific config object, rather than being ignored globally.

Using `globalIgnores` avoids that potential mistake and is more explicit.